app/Http/Controllers/Api_key/DashboardController.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
the normal pattern for Controller paths is PascalCase. Please change "Api_key" to "ApiKey"